packages/runtime/container-runtime/src/test/dataStoreContext.spec.ts

```diff
@@ -244,6 +241,8 @@ describe("Data Store Context Tests", () => {
 					dataStoreAttributes.isRootDataStore,
 					"Local DataStore root state does not match",
 				);
+				const type =
```

Comment:
Probably worth adding a free function somewhere to do this, it's worth naming and codifying in one place IMO.